pkg/kubelet/cadvisor/cadvisor_linux.go

```diff
@@ -96,7 +92,7 @@ func containerLabels(c *cadvisorapi.ContainerInfo) map[string]string {
 }
 
 // New creates a cAdvisor and exports its API on the specified port if port > 0.
-func New(address string, port uint, runtime string, rootPath string) (Interface, error) {
+func New(address string, port uint, statscacheDuration, maxHousekeepingInterval metav1.Duration, runtime string, rootPath string) (Interface, error) {
```

Comment:
`allowDynamicHousekeeping` is still not configurable. It is still using default constant value.